data/json/tool_qualities.json

```diff
@@ -176,6 +176,12 @@
     "id": "SELF_JACK",
     "name": { "str": "self jacking" }
   },
+  {
+    "type": "tool_quality",
+    "id": "HOSE_SIPHON",
```

Comment:
I would probably name this quality just `HOSE`.

Reply:
Done.